optimizely/event/event_processor.py

```diff
@@ -222,3 +222,29 @@ def dispose(self):
       return
 
     self._disposed = True
+
+
+class ForwardingEventProcessor(EventProcessor):
+
+  def __init__(self, event_dispatcher, logger, notification_center=None):
+    self.event_dispatcher = event_dispatcher
+    self.logger = logger
```

Comment:
What if no logger or dispatcher is provided.

Reply:
Providing default_event_dispatcher and NoOpLogger as default values.